lib/airbrake.js

```diff
@@ -50,59 +48,61 @@ function Airbrake() {
 
 _.assign(Airbrake.prototype, EventEmitter.prototype);
 
-Airbrake.PACKAGE = (function() {
+Airbrake.PACKAGE = (function () {
   var json = fs.readFileSync(__dirname + '/../package.json', 'utf8');
   return JSON.parse(json);
-})();
+}());
 
-Airbrake.createClient = function(projectId, key, env) {
+Airbrake.createClient = function (projectId, key, env) {
   var instance = new this();
   instance.key = key;
   instance.env = env || instance.env;
   instance.projectId = projectId || instance.projectId;
   return instance;
 };
 
-Airbrake.prototype.expressHandler = function(disableUncaughtException) {
+Airbrake.prototype.expressHandler = function (disableUncaughtException) {
   var self = this;
 
-  if(!disableUncaughtException) {
-    process.on('uncaughtException', function(err) {
+  if (!disableUncaughtException) {
+    process.on('uncaughtException', function (err) {
       self._onError(err, true);
     });
   }
 
   return function errorHandler(err, req, res, next) {
-    if (res.statusCode < 400) res.statusCode = 500;
+    var error = err;
+    var requestObj = req;
+    var responseObj = res;
 
-    err.url = req.url;
-    err.component = req.url;
-    err.action = req.method;
-    err.params = req.body;
-    err.session = req.session;
-    err.ua = req.get('User-Agent');
+    if (responseObj.statusCode < 400) responseObj.statusCode = 500;
+
+    error.url = requestObj.url;
+    error.component = requestObj.url;
+    error.action = requestObj.method;
+    error.params = requestObj.body;
+    error.session = requestObj.session;
+    error.ua = requestObj.get('User-Agent');
 
     self._onError(err, false);
     next(err);
   };
 };
 
-Airbrake.prototype._onError = function(err, die) {
+Airbrake.prototype._onError = function (err, die) {
   var self = this;
-  if (!(err instanceof Error)) {
-    err = new Error(err);
-  }
+  var error = (err instanceof Error) ? err : new Error(err);
```

Comment:
While other node style guides suggest using multi-lines for ternaries the AirBNB guide prefers a [single line](https://github.com/airbnb/javascript#comparison--nested-ternaries) ternary.

:+1: for readability and being consistent.